### DISCUSSION TITLE
Problem with CCSID 297 or 1147 (french) ?

### DISCUSSION THREAD

**Discussion by ThomasBarberot** [codefori/vscode-ibmi#1032] [Q&A] [answered]
Hi.
I'm french, so I work with accented character (like "é" or "à"). I think I have the correct connexion settings, but sources are incorrectly saved, with "unreadable" characters.

My profile on IBM i is with
- LANGID(*SYSVAL)
- CNTRYID(*SYSVAL)
- CCSID(*SYSVAL)
- CHRIDCTL(*SYSVAL)

System values on IBM i are set like this : 
- QLNGID : FRA
- QCNTRYID : FR
- QCCSID : 297
- QCHRIDCTL : *DEVD
- and QCHRID : 697 / 297

Source files have CCSID 297 (I also tried with SRCPF with CCSID 1147 and 65535, it doesn't work)

Connexion setting to IBM i in VSC is with Source file CCSID : *FILE

And when I try to reload the source membre changing the CCSID, I don't find french CCSID in the list
Did I miss something ? Should I have to load a configuration pack for french CCSID ?
Is it impossible ? How french people does ?

Thanks a lot.

**Comment by worksofliam** [comment, marked as answer]
We actually have lots of French users with no issues. Perhaps your SSH environment is not setup correctly.

Check out this page in our docs: https://halcyon-tech.github.io/docs/#/pages/tips/ccsid

**Reply by ThomasBarberot** [reply]
The pase configuration was incorrect. I followed instructions on your page, I added environement variables and it's good. Thanks

**Reply by worksofliam** [reply]
Glad to hear it!!
